src/AppBundle/Entity/User.php

```diff
@@ -63,12 +72,62 @@ public function getPlants()
     /**
      * Set plantSpecification
      *
+     * @param \PlantBundle\Entity\Plant $plant
+     *
      * @param array $plants
```

Comment:
now this is `addPlant($plant)` so PHPDoc annotation is no longer valid (array $plants given)